javaparser-core/src/main/java/com/github/javaparser/ast/CompilationUnit.java

```diff
@@ -72,6 +73,13 @@ public CompilationUnit(int beginLine, int beginColumn, int endLine, int endColum
         setTypes(types);
     }
 
+	public CompilationUnit(Position begin, Position end, PackageDeclaration pakage, List<ImportDeclaration> imports, List<TypeDeclaration> types) {
```

Comment:
Copy-pasted names from the other constructors. I did not want to introduce inconsistancies and modify stuff outside of my current task (as too not confuse the diffs).

Well blame the previous coder!! :P
But I'd be happy to make the change in an other commit!